pcsx2/R3000A.cpp

```diff
@@ -63,8 +63,8 @@ void psxReset()
 	g_iopNextEventCycle = psxRegs.cycle + 4;
 
 	psxHwReset();
-	psxEEmultiplier = 8;
-	PSXCLK = 36864000;// to reset to default ps2 mode speed if someone decides to switch from ps1 to 2.
+	psxEEmultiplier = 8; // to reset to default ps2 mode speed if someone decides to switch from ps1 to 2.
+	PSXCLK = 36864000; // to reset to default ps2 mode speed if someone decides to switch from ps1 to 2.
 	ioman::reset();
```

Comment:
Just a note for when you next look at this, if you could move these 2 lines above the reset functions (maybe below the memzero(psxRegs) line), just for neatness, that'd be cool :)

Reply:
Will do.